cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -479,13 +479,13 @@ func (p *BicepProvider) purgeItems(
 		if err != nil {
 			return err
 		}
-
-		for _, item := range items {
-			if err := item.purge(); err != nil {
-				return fmt.Errorf("failed to purge %s: %w", item.resourceType, err)
-			}
+	}
+	for _, item := range items {
```

Comment:
Let's remember to call this out in the release notes as an issue we fixed, because it is user facing.